docs/modules/mysql.md

```diff
@@ -1,9 +1,10 @@
 # MySQL
 
 ### Purpose
+[MySQL](https://www.mysql.com/) is a database management system. MySQL will organize and provide access to databases where sites can store information.
 
 ### Dependencies
-
+MySQL can be run on Windows, OSX, RHEL/CentOS, Ubuntu, and Fedora. MySQL will require PHP5. These packages can also be downloaded from APT by running `sudo apt-get install mysql-server php5-mysql`. Once the packages are installed we need to create a database directory structure by typing `sudo mysql_install_db`. For more information, please refer to the [documentation](https://dev.mysql.com/doc/). 
```

Comment:
I was a  on the fence when I was detailing the install instructions. Thanks for the feedback! I will remove it.